doc/whats_new/v0.21.rst

```diff
@@ -2,6 +2,23 @@
 
 .. currentmodule:: sklearn
 
+.. _changes_0_21_2:
+
+Version 0.21.2
+==============
+
+**June 2019**
+
+Changelog
+---------
+
+:mod:`sklearn.metrics`
+......................
+
+- |Fix| Fixed a bug in :func:`euclidean_distances` where a part of the distance
```

Comment:
Maybe mention that is a regression in 0.20.1, not an earlier bug that was fixed.

Reply:
It was introduced in 0.21.0 I think.

Reply:
@gokceneraslan yes, the below diff notes correctly "(regression introduced in 0.21)".